### ISSUE TITLE
Automatic1111 1.6 Adds Default Brush Color Option

### ISSUE THREAD

**Issue by 66e61zhv** [richrobber2/canvas-zoom#94]
The UI now has its own native brush color option under Settings > img2img.

Just FYI in case other people were wondering what happened to their default brush colour.

**Comment by daswer123** [comment]
Yes I know and have already added a warning in canvas-zoom settings and removed the extension's impact on this feature
